iati/core/tests/test_resources.py

```diff
@@ -1,6 +1,7 @@
 """A module containing tests for the library implementation of accessing resources."""
 from lxml import etree
 import pytest
+from six import string_types
```

Comment:
As a result of discussion this morning, it was agreed within the team to use the `six` module, but refer to it as `six.string_types`. This will help identify instances of the module if compatibility problems are later identified.